packages/ui/src/index.tsx

```diff
@@ -18,7 +18,7 @@ const uiConfig = JSON.parse(
   document.getElementById('__UI_CONFIG__')?.textContent || '{}'
 ) as UIConfig;
 
-initI18n({ lng: uiConfig.locale?.lng || 'en-US', basePath }).then(() => {
+initI18n({ lng: uiConfig.locale?.lng || navigator.language || 'en-US', basePath }).then(() => {
```

Comment:
This may break translations since we support only 3 languages

Reply:
https://github.com/felixmosh/bull-board/blob/8856fb5277f43b8bbe3be10289c54a3e9a99d180/packages/ui/src/services/i18n.ts#L22

What I understand about this configuration is that if the incoming language does not exist, en-US will be used instead, right?

Reply:
Yeap, correct...